pex/variables.py

```diff
@@ -44,6 +45,17 @@ def delete(self, variable):
   def set(self, variable, value):
     self._environ[variable] = str(value)
 
+  def _from_rc(self, rc):
+    ret_vars = {}
+    for filename in [rc, '.pexrc']:
```

Comment:
Yeah, I figured if someone has a ~/.pexrc, they might want those variables to be overwritten by the .pexrc that sits alongside their pex.

The order of precedence is:

~/.pexrc
relative .pexrc
environment variables